### DISCUSSION TITLE
is peak matrix extracted with getMatrixFromProject normalized?

### DISCUSSION THREAD

**Discussion by Brawni** [GreenleafLab/ArchR#943] [Questions / Documentation] [answered]
Hello,

Quick question, is the peak matrix obtained with `getMatrixFromProject()` normalized by any means ? I looked up the code and it seems that it should contain the raw insertion count per cell per peak capped with ceiling but I just wanted to be 100% sure. 

Thank you!

**Comment by rcorces** [comment, marked as answer]
correct, raw insertion count taking into account whatever was used for `ceiling` when the matrix was added to the project

**Reply by Brawni** [reply]
great thanks for the quick reply! Might be worth specify that in the function doc.

Thanks!

**Reply by rcorces** [reply]
Well, I think its more complex then my quick answer. It gives whatever matrix you ask it for, without altering that matrix unless you tell it to. For example, if you added your `PeakMatrix` using `addPeakMatrix()` with `binarize = TRUE`, then `getMatrixFromProject()` will return a binarized `PeakMatrix`. Alternatively, you could set `binarize = TRUE` in `getMatrixFromProject()` and binarize the `PeakMatrix` as you pull it out. Similar issues with `GeneScoreMatrix` which can have various scaling factors etc applied to it during creation. I dont think this is easily clarified in the function docs - the function just gets you the matrix in whatever format it is currently.

**Reply by rcorces** [reply]
I did my best to update the function explanation. Feel free to provide input
https://github.com/GreenleafLab/ArchR/commit/79f567e0b038cb49d01626fd504ed41e3c85ce46

**Reply by Brawni** [reply]
I see. However as per default the peak matrix you get is not normalized, I think this is important to be reminded and thanks for adding that in the docs!
